### DISCUSSION TITLE
verse_onprem/3.2_test/user/welcometoibmverse

### DISCUSSION THREAD

**Discussion by giscus** [HCL-TECH-SOFTWARE/HCLSoftware-Documentation-Home#78] [Q&A]
# verse_onprem/3.2_test/user/welcometoibmverse

HCL Verse is mail that understands you. It has less clutter and more clarity for getting you connected to the people who matter to you most.

https://help.hcl-software.com/verse_onprem/3.2_test/user/welcometoibmverse.html

**Comment by syeda-git** [comment]
Test comment - Syed
